src/components/styles/Form.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: 'Montserrat';
+  src: url('https://fonts.googleapis.com/css?family=Montserrat') format('truetype');
+}
+
+.input-form {
+  display: flex;
+  flex-direction: row;
+}
+
+.form-title {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 2.813rem 32.75rem 0.188rem 3.5rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
- [OPTIONAL] As `normal` is the default values and I can see that you are writing similar lines in all of your CSS files. Actually, you have copied it from the zeplin. there is no problem is that but you should remove these lines as they are redundant and against DRY principle.